concordium-consensus/src/Concordium/GlobalState/Persistent/BlockState/ProtocolLevelTokens.hs

```diff
@@ -452,3 +457,83 @@ migrateProtocolLevelTokensForPV migration oldPLTsPV@(ProtocolLevelTokensForPV (C
         SFalse ->
             -- There are no migrations that remove PLTs altogether.
             case migration of {}
+
+-- | The table of PLTs account states. The table is indexed by the token index
+--  into the global token table.
+newtype TokenAccountStateTable = TokenAccountStateTable
```

Comment:
The account related persisted data structures should find a better home. Maybe in `Concordium/Globalstate/Accounts.hs`? But I can't find a solution without running into cyclic imports.